addons/l10n_ph/tests/test_bir_2307_generation.py

```
@@ -58,9 +58,9 @@ def test_bir_2307_company(self):
         # 2: Build the expected values
         expected_values = {
             # Header
-            0: ['Reporting_Month', 'Vendor_TIN', 'branchCode', 'companyName', 'surName', 'firstName', 'middleName', 'address', 'nature', 'ATC', 'income_payment', 'ewt_rate', 'tax_amount'],
+            0: ['Reporting_Month', 'Vendor_TIN', 'branchCode', 'companyName', 'surName', 'firstName', 'middleName', 'address', 'zip_code', 'nature', 'ATC', 'income_payment', 'ewt_rate', 'tax_amount'],
             # Row
-            1: ['01/15/2020', '789456123', '789', 'Test Partner', '', '', '', '9 Super Street, Super City, Philippines', 'Test line', 'WC640', 100.0, 1.0, 1.0],
+            1: ['01/15/2020', '789456123', '789', '', '', '', '', '9 Super Street, Super City, Philippines', '8888', '1% WC640 - Supplier of goods', 'WC640', 100.0, 1.0, 1.0],
```

There is neither a `companyName` nor the `surName` / `firstName` / `middleName` on this line.
Such cases are also in the `enterprise` branch.
I guess it is because it is the person but the special `first_name`, `middle_name`, `last_name` fields are not set.
Maybe we should raise an error in such cases?
I think we should at least adjust the tests to give only "complete" output.